core/arch/arm/plat-imx/platform_config.h

```diff
@@ -158,18 +101,26 @@
 #if defined(PLATFORM_FLAVOR_mx6dlsabresd)
 #define CONSOLE_UART_BASE		UART1_BASE
 #endif
-#define DRAM0_BASE			0x10000000
-#define DRAM0_SIZE			0x40000000
-
-#define CFG_TEE_RAM_VA_SIZE		(1024 * 1024)
 
+/* Board specific RAM size */
 #if defined(PLATFORM_FLAVOR_mx6qsabrelite) || \
-	defined(PLATFORM_FLAVOR_mx6qsabresd)
+	defined(PLATFORM_FLAVOR_mx6qsabresd) || \
+	defined(PLATFORM_FLAVOR_mx6dlsabresd)
+#define DRAM0_SIZE			0x40000000
+#endif
+
+/* Core number depends of SoC version. */
+#if defined(CFG_MX6Q)
 #define CFG_TEE_CORE_NB_CORE		4
-#else
+#elif defined(CFG_MX6D) || defined(CFG_MX6DL)
 #define CFG_TEE_CORE_NB_CORE		2
+#else
```

Comment:
minor: Since the number of supported platforms, it seems not clear what this last `else` expect. I would rather explicitly state some `if defined(...)`.